src/wallet/rpcwallet.cpp

```diff
@@ -5193,6 +5195,7 @@ UniValue z_sendmany(const UniValue& params, bool fHelp)
         if (std::holds_alternative<libzcash::UnifiedAddress>(decoded.value())) {
             ua = std::get<libzcash::UnifiedAddress>(decoded.value());
             involvesUnifiedAddress = true;
+            involvesOrchard = involvesOrchard || ua.value().GetOrchardReceiver().has_value();
```

Comment:
So, this is maybe a bit pendantic of me, but I prefer to use the boolean operator to the bitwise operator because the bitwise `or` represents an operation on too large of a state space (8 bits instead of 1 - that's 2^7 too many states!).